invenio/modules/relationships/models.py

```diff
@@ -0,0 +1,163 @@
+# -*- coding: utf-8 -*-
+##
+## This file is part of Invenio.
+## Copyright (C) 2015 CERN.
+##
+## Invenio is free software; you can redistribute it and/or
+## modify it under the terms of the GNU General Public License as
+## published by the Free Software Foundation; either version 2 of the
+## License, or (at your option) any later version.
+##
+## Invenio is distributed in the hope that it will be useful, but
+## WITHOUT ANY WARRANTY; without even the implied warranty of
+## MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+## General Public License for more details.
+##
+## You should have received a copy of the GNU General Public License
+## along with Invenio; if not, write to the Free Software Foundation, Inc.,
+## 59 Temple Place, Suite 330, Boston, MA 02111-1307, USA.
+
+"""Database model for persisting relationships."""
+
+from uuid import uuid4
+
+from sqlalchemy_utils import UUIDType
+
+from invenio.ext.sqlalchemy import db, utils
+from invenio.modules.records.models import RecordMetadata
+
+from .errors import NoSuchRecordException, WrongModelNameException
+
+
+class Relationship(db.Model):
+
+    """Represent a graph edge."""
+
+    uuid = db.Column(UUIDType(binary=False), primary_key=True)
+    model_from = db.Column(db.String(255), index=True)
```

Comment:
What is `model_from`/`model_to`? Aren't we going to link records all the time? If so, isn't the model already a property of the record?